packages/dds/sequence/api-extractor.json

```diff
@@ -1,4 +1,18 @@
 {
 	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
-	"extends": "@fluidframework/build-common/api-extractor-common-strict.json"
+	"extends": "@fluidframework/build-common/api-extractor-base.json",
+
+	// Configures how the .d.ts rollup file will be generated.
+	"dtsRollup": {
+		// This package has a root module named "sequence" which conflicts with the default naming here.
```

Comment:
I've opened #17740 to make this sort of collision less likely to occur in the future. I'll update the comment here to note that this override can be removed once a future version of `build-common` has been integrated.